server/container_create_linux.go

```diff
@@ -268,7 +268,12 @@ func (s *Server) createSandboxContainer(ctx context.Context, ctr ctrIface.Contai
 		processLabel = ""
 	}
 
-	containerVolumes, ociMounts, err := addOCIBindMounts(ctx, mountLabel, containerConfig, specgen, s.config.RuntimeConfig.BindMountPrefix, s.config.AbsentMountSourcesToReject)
+	var cgroupRW bool;
+	if node.CgroupIsV2() && sb.Annotations()[crioann.CgroupRWAnnotation] == "true" {
+		cgroupRW = true;
+	}
```

Comment:
we could compress this to 
`cgroupRW := node.CgroupIsV2() && sb.Annotations()[crioann.CgroupRWAnnotation] == "true"`